source/class/cv/parser/pure/widgets/ColorChooser.js

```diff
@@ -39,40 +38,54 @@ qx.Class.define('cv.parser.pure.widgets.ColorChooser', {
+        this.getAttributeToPropertyMappings()
+      );
 
-      data.baseColors = { // default to sRGB color space with D65 white point
-        r: {x: 0.64, y: 0.33, Y: 0.2126},
-        g: {x: 0.30, y: 0.60, Y: 0.7152},
-        b: {x: 0.15, y: 0.06, Y: 0.0722},
-        w: {x: 0.3127, y: 0.3290, Y: 1}
+      cv.parser.pure.WidgetParser.parseFormat(xml, path);
+      cv.parser.pure.WidgetParser.parseAddress(
+        xml,
+        path,
+        this.makeAddressListFn
+      );
+
+      data.baseColors = {
+        // default to sRGB color space with D65 white point
+        r: { x: 0.64, y: 0.33, Y: 0.2126 },
+        g: { x: 0.3, y: 0.6, Y: 0.7152 },
+        b: { x: 0.15, y: 0.06, Y: 0.0722 },
+        w: { x: 0.3127, y: 0.329, Y: 1 }
       };
-      let r_x          = xml.getAttribute('r_x');
-      let r_y          = xml.getAttribute('r_y');
+
+      let r_x = xml.getAttribute('r_x');
+      let r_y = xml.getAttribute('r_y');
       let r_wavelength = xml.getAttribute('r_wavelength');
-      let r_strength   = xml.getAttribute('r_strength');
-      let r_curve      = xml.getAttribute('r_curve');
-      let r_scale      = xml.getAttribute('r_scale');
-      let g_x          = xml.getAttribute('g_x');
-      let g_y          = xml.getAttribute('g_y');
+      let r_strength = xml.getAttribute('r_strength');
+      let r_curve = xml.getAttribute('r_curve');
+      let r_scale = xml.getAttribute('r_scale');
+      let g_x = xml.getAttribute('g_x');
+      let g_y = xml.getAttribute('g_y');
       let g_wavelength = xml.getAttribute('g_wavelength');
-      let g_strength   = xml.getAttribute('g_strength');
-      let g_curve      = xml.getAttribute('g_curve');
-      let g_scale      = xml.getAttribute('g_scale');
-      let b_x          = xml.getAttribute('b_x');
-      let b_y          = xml.getAttribute('b_y');
+      let g_strength = xml.getAttribute('g_strength');
+      let g_curve = xml.getAttribute('g_curve');
+      let g_scale = xml.getAttribute('g_scale');
+      let b_x = xml.getAttribute('b_x');
+      let b_y = xml.getAttribute('b_y');
       let b_wavelength = xml.getAttribute('b_wavelength');
-      let b_strength   = xml.getAttribute('b_strength');
-      let b_curve      = xml.getAttribute('b_curve');
-      let b_scale      = xml.getAttribute('b_scale');
-      let w_x          = xml.getAttribute('w_x');
-      let w_y          = xml.getAttribute('w_y');
-      let w_strength   = xml.getAttribute('w_strength');
-      let w_curve      = xml.getAttribute('w_curve');
-      let w_scale      = xml.getAttribute('w_scale');
+      let b_strength = xml.getAttribute('b_strength');
+      let b_curve = xml.getAttribute('b_curve');
+      let b_scale = xml.getAttribute('b_scale');
+      let w_x = xml.getAttribute('w_x');
+      let w_y = xml.getAttribute('w_y');
+      let w_strength = xml.getAttribute('w_strength');
+      let w_curve = xml.getAttribute('w_curve');
+      let w_scale = xml.getAttribute('w_scale');
```

Comment:
Is there an option to keep the `=` aligned?

Reply:
No

Reply:
These are all available options: https://prettier.io/docs/en/options.html